packages/tools/pluggable-widgets-tools/configs/rollup.config.native.js

```diff
@@ -194,6 +195,7 @@ export default async args => {
     function getClientComponentPlugins() {
         return [
             isTypescript ? widgetTyping({ sourceDir: join(sourcePath, "src") }) : null,
+            execShellCommand(`npx pluggable-widgets-tools format`, sourcePath),
```

Comment:
The PR description does not mention anything about formatting. Why do we need this step? Don't we already have Prettier configured by default as a pre-release step? To me, it feels rather intrusive for customers if the pluggable widget tools start formatting unconditionally without an easy way to override it.

Reply:
Sorry I forgot to update the description. Lots of things have changed regarding rollup lately.

The reason for this step is that our generated type files are not pretty. This results in a lot of unwanted changes and means the developer has to either discard them, or include them in their commit (so it's fixed by a pre-commit hook which includes prettier).
I don't like either of those "solutions" (discard / pre-commit). Therefore I added this format step.
I can see that it could be intrusive for customers. Maybe we should add another env variable for our internal use, all though I imagine a lot of customers are running into the same issue if they prettify their codebase. What do you think would be the best solution?

Reply:
I couldn't find Isa's solution, but I've updated the code so it only targets the `typings/` folder.

Reply:
https://github.com/mendix/widgets-resources/pull/941/files

Reply:
I've also suggested in that PR that this should probably be part of the `widgetTyping` plugin itself, in which case you would not need a separate solution here. In general, this kind of behaviour should be kept in sync between web and native, so that would be a good way to achieve that :).

Reply:
I agree, although I didn't expect web to apply a similar solution in the end. One of the reasons why we split rollup is because native wanted this feature while web did not.. 😕 

Anyhow I updated my code so formatting is now part of `widgetTyping` 👍🏼

Reply:
Did you coordinate with Isa? Because if he now tries to do the same thing, one of you will get a merge conflict on the `widgetTyping` plugin.

I think that web definitely did not want a reformat on the entire project, which is what happened before :). I can see other advantages in splitting up the rollup config for web and native, but we definitely shouldn't introduce arbitrary divergence between the two.

Reply:
Everything from #941 is taken care of in this PR now.